### PR TITLE
fix: Fix UI of search event items

### LINKED ISSUE #339: Reduce the space between any two events in Search fragment and improve UI
Working on it :+1:

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #339 
screenshot_20180720-131744

### CODE REVIEW

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -26,7 +27,13 @@
     <!-- card details -->
     <dimen name="card_elevation">3dp</dimen>
     <dimen name="card_corner_radius">3dp</dimen>
+    <dimen name="image_corner_radius">6dp</dimen>
     <dimen name="avatar_medium">120dp</dimen>
+    <dimen name="details_margin_small">4dp</dimen>
+    <dimen name="details_text_size">13sp</dimen>
+    <dimen name="image_card_specific">100dp</dimen>
+    <dimen name="event_name_text_size">19sp</dimen>
+    <dimen name="details_box_margin">10dp</dimen>
```
Comment (iamareebjamal) [inline app/src/main/res/values/dimens.xml]: These are too specific dimens

Comment (dreadpool2) [reply app/src/main/res/values/dimens.xml]: Should I just use them as it is?

Comment (dreadpool2) [reply app/src/main/res/values/dimens.xml]: PR Updated

**Comment by dreadpool2** [comment]
@nikit19 Please merge this :+1:

**Pull request merged**